gen3-client/g3cmd/utils.go

```diff
@@ -57,13 +58,15 @@ type MultipartUploadRequestObject struct {
 	Key        string `json:"key"`
 	UploadID   string `json:"uploadId"`
 	PartNumber int    `json:"partNumber"`
+	Bucket *string `json:"bucket,omitempty"`
```

Comment:
Same comment here. The pattern for other variables here seem to be to use the raw type (string, int) rather than a pointer